test/zdtm/static/timens_nested.c

```diff
@@ -10,7 +10,8 @@
 
 #include "zdtmtst.h"
 
-const char *test_doc = "Check nested time namespaces";
+const char *test_doc = "Check ability to restore nested time namespace\n"
+	"task that has been moved to a different namespace.";
```

Comment:
code linter error:
```diff
-	"task that has been moved to a different namespace.";
+		       "task that has been moved to a different namespace.";
```

Reply:
Fixed the indentation in my latest commit